app/presenters/transition_landing_page_presenter.rb

```diff
@@ -66,7 +66,7 @@ def fetch_comms
       data_attributes = {
         track_category: "transition-landing-page",
         track_action: link_item[:link][:path],
-        track_label: link_item[:link][:text],
+        track_label: "News",
```

Comment:
This seems like an unusual use for the label; do we know we we're not just changing the category?